src/modules/site-v2/base/views/tools/phenotype_database.py

```diff
@@ -67,21 +67,34 @@ def results_columns():
 def phenotype_database():
   """
     Phenotype Database table (non-bulk)
-  """
+  """  
   # Get the list of traits for non-bulk files
   try:
-    traits_non_bulk = query_phenotype_metadata()
+    page = request.args.get('page', 1, type=int)
+    per_page = 10
+    query = query_phenotype_metadata()
+
+    # Get the list of unique tags
+    tags = [ tr.tags.split(', ') for tr in query if tr.tags ]
+    tags_list = [tg for tr_tag in tags for tg in tr_tag]
+    unique_tags = list(set(tags_list))
```

Comment:
Love the use of `set` and `list` to handle duplicates! :)

Do we need to sort the list of tags before rendering, e.g. alphabetically?  (If not, or if you already do that elsewhere, discard this msg)